jquery.flot.composeImages.js

```diff
@@ -38,10 +38,27 @@
     }
 
     function getGenerateTempImg(tempImg, canvasOrSvgSource) {
+        tempImg.componentName = canvasOrSvgSource.className + '  (' + canvasOrSvgSource.tagName + ')';
```

Comment:
Maybe *sourceDescription* instead of *componentName*?

Reply:
I think it will be nice to reuse the HTML format for *sourceDescription*:
```
<tag class="..." id="..." />
```
Where *tag* is the tag name of the source (SVG or CANVAS).